README.md

```diff
@@ -134,30 +132,28 @@ tx1 = models.TransactionRequest(
     source="Broker"
 )
 
-tx_portfolios_api.upsert_transactions(scope=scope, code=portfolio_code, transaction_request=[tx1])
+await tx_portfolios_api.upsert_transactions(scope=scope, code=portfolio_code, transaction_request=[tx1])
 ```
 
 #### Get holdings
 
 ```python
 tx_portfolios_api = api_factory.build(lusid.api.TransactionPortfoliosApi)
 
-holdings_response = tx_portfolios_api.get_holdings(
+holdings_response = await tx_portfolios_api.get_holdings(
     scope=scope, code=portfolio_code, property_keys=["Instrument/default/Name"]).values
 
 print("Holdings:")
 for holding in holdings_response:
     print(luid_to_name[holding.instrument_uid], holding.units, holding.cost.amount)
 ```
 
```

Comment:
i wonder if we an a snippet for creating the sync version too?

Reply:
I think that is better elsewhere - its not supposed to be the default way to use the SDK, its mainly provided for backwards compatibility

Reply:
👍